src/hardware/innovation.cpp

```diff
@@ -129,16 +126,18 @@ void Innovation::Open(const std::string_view model_choice,
 	// Ready state-values for rendering
 	last_rendered_ms = 0.0;
 
+	// Variable model_name is only used for logging, so use a const char* here
+	const char* model_name{model_choice == "8580" ? "8580" : "6581"};
```

Comment:
Minor: We try not to use the `var_name{...}` initialiser syntax.

Reply:
Fixed.